build-scripts/hack/update-component-versions.py

```diff
@@ -36,6 +37,14 @@
 # Helm release branch to track. The most recent tag in the branch will be used.
 HELM_RELEASE_BRANCH = "release-3.14"
 
+# MetalLB Helm repository and chart version
+METALLB_REPO = "https://metallb.github.io/metallb"
+METALLB_CHART_VERSION = "0.14.5"
+
+# Metrics Server Helm repository
+METRICS_REPO = "https://kubernetes-sigs.github.io/metrics-server"
```

Comment:
NOTE: these are not relevant for this PR.

Reply:
Please revert this, and also remove the `metrics-server-3.12.1.tgz` chart. This PR is related to adding MetalLB support, not changing metrics-server.